fe/fe-core/src/main/java/com/starrocks/common/FeConstants.java

```diff
@@ -65,4 +65,6 @@ public class FeConstants {
     public static boolean USE_MOCK_DICT_MANAGER = false;
 
     public static final int default_tablet_number = 128;
+    //predicate reorder
+    public static boolean OPEN_PREDICATE_REORDER = true;
```

Comment:
Should use session variable, not constant variable.

Reply:
My mentor @Seaven suggest use constant variable, May be we need to discuss.

Reply:
Use session variable now, default  status is disabled